managed/services/checks/checks.go

```diff
@@ -1490,24 +1490,24 @@ func (s *Service) filterSupportedChecks(advisors []check.Advisor) []check.Adviso
 	for _, advisor := range advisors {
 		checks := make([]check.Check, 0, len(advisor.Checks))
 
-	loop:
+	LOOP:
```

Comment:
Effective go uses just capitalized words for labels: https://go.dev/doc/effective_go#switch